Automattic/themes
# Course Theme: Add gravatar hovercard in comments

#### Changes proposed in this Pull Request:

- Added gravatar hovercard to comment query block

#### Gravatar hovercard library documentation
https://github.com/gravatar/hovercards/blob/0.5.8/README.md

#### Testing instructions:

- Enable this branch of course theme
- Enable comments on a page/post
- Go to the page/post from frontend
- Comment as a user who's email has a gravatar account
- Hover on the gravatar image you see in the comment
- Make sure you see the hovercard

https://github.com/Automattic/themes/assets/6820724/449f4f86-e996-45cf-b199-1c6c10e375c2

#### Related issue(s):
https://github.com/Automattic/themes/issues/7306
